src/server/disabled.ts

```diff
@@ -1,14 +1,11 @@
 onEvent("recipes", (event) => {
   // Disable some mods temporarily
-  event.remove({ mod: "theoneprobe" });
+  //event.remove({ mod: "theoneprobe" });
   event.remove({ mod: "sophisticatedstorage" });
   event.remove({ mod: "functionalstorage" });
   event.remove({ mod: "prettypipes" });
-  event.remove({ mod: "torchmaster" });
-  event.remove({ mod: "chunkloaders" });
-
-  // Temporarily disable Warp Stone
-  event.remove({ output: "waystones:warp_stone" });
+  //event.remove({ mod: "torchmaster" });
+  //event.remove({ mod: "chunkloaders" });
```

Comment:
remove these comments and remove the actual recipes for the items you had hidden in `client/disabled.ts`